03LessonThree/in-class-project/daily-planet.html

```diff
@@ -5,6 +5,32 @@
         <link rel="stylesheet" href="css/daily-planet-style.css"/>
     </head>
     <body>
-        <!-- You code start here -->
+      <header>
+        <h1>THE DAILY PLANET</h1>
+      </header>
+        <nav>
+          <a href="http://espn.go.com/">SPORTS</a>
+          <a href="http://weather.com/">WEATHER</a>
+          <a href="http://www.austinchronicle.com/">LOCAL</a>
+        </nav>
+      <section>
+        <h1>Trump witholds support</h1>
+        <img src="http://i2.cdn.turner.com/cnnnext/dam/assets/160114170540-ryan-trump-mccain-split-overlay-tease.jpg">
+        <span><b>Washington (CNN) -</b> Donald Trump is declining to endorse a pair of Republican heavyweights --
+          House Speaker Paul Ryan and former presidential nominee John McCain -- in their primary fights, another
+          sign of the fractures within the party.
+          <p>Ryan and McCain face long-shot challengers in their respective primary races in Wisconsin and Arizona. Yet their party's presidential nominee would not bless them in an interview with The Washington Post on Tuesday.
+          <p>"I like Paul, but these are horrible times for our country. We need very strong leadership. We need very, very strong leadership. And I'm just not quite there yet. I'm not quite there yet," Trump said of Ryan.
+        </span>
+      </section>
+      <aside>
+        <img src="http://orig00.deviantart.net/9f66/f/2009/213/4/c/long_cat_is_long_by_plastik_rainbows.jpg">
+      </aside>
+      <footer>
+        This is my foot

```

Comment:
Don't forget to add the project requirement here:

"Add copyright of the company and the current year."

If you can add this and commit again, I can bump your grade to 100.